src/igvfd/schemas/changelogs/rodent_donor.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`rodent_donor.json`*
 
+### Schema version 3
+
+* Rename `resource_identifier` to `donor_identifier` and linked `resource_name` to `Source` in `external_resource` from base class `donor`.
```

Comment:
Minor grammar comment: change "linked" to "link"
Since we have capitalized the schema title "Source" then I think it makes sense to also capitalize "Donor".